devtools/platform-descriptor-json-plugin/src/main/java/io/quarkus/maven/GenerateExtensionsJsonMojo.java

```diff
@@ -135,6 +143,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         String quarkusCoreVersion = null;
         for (Dependency dep : deps) {
             final Artifact artifact = dep.getArtifact();
+            if (ignoredGroupIds.contains(artifact.getGroupId())) {
```

Comment:
Looks fine to me. Just one question - do the Maven mojos have their `Set` type members initialized to a non-null instance if no configurations have been specified for that parameter? If not, I think we should include a null check here for `ignoredGroupIds` just to prevent a NPE in future when/if we stop configuring any ignored groups.

Reply:
Good catch, turns out it doesn't